src/content/en/2019/javascript.md

```diff
@@ -342,7 +338,7 @@ If native modules are used, it's important to ensure that an appropriate fallbac
 
 When used together, browsers that support modules will completely ignore any scripts containing the `nomodule` attribute. On the other hand, browsers that do not yet support modules will not download any scripts with `type="module"`. Since they do not recognize `nomodule` either, they will download scripts with the attribute normally. Using this approach can allow developers to [send modern code to modern browsers for faster page loads](https://web.dev/serve-modern-code-to-modern-browsers/).
 
-So, **how many sites use `nomodule` for scripts on their page?**
+So, how many sites use `nomodule` for scripts on their page?
```

Comment:
In other places, you removed the breakline before the question introducing the graph. Why did you leave it here?

Reply:
I didn't remove them - the newlines didn't exist, and I just removed the bolding of the sentence. But it's a fair question and I think they should be removed for consistency.